parsec/backend/asgi/rpc.py

```diff
@@ -1,10 +1,13 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
-from __future__ import annotations
 
-from base64 import b64decode
-from typing import NoReturn, Tuple
+from __future__ import annotations
+import trio
+from base64 import b64decode, b64encode
+from typing import AsyncIterator, AsyncIterable, NoReturn, Tuple, Optional, Any
```

Comment:
`Optional` has been replaced everywhere in the codebase, use `X | None` instead

Reply:
done.